### ISSUE TITLE
Feature request: Instructions to build single HTML file?

### ISSUE THREAD

**Issue by rickhg12hs** [gchq/CyberChef#562]
### How to transform downloadable ZIP file into single HTML file?

Though large, sharing a single HTML file has some advantages ... It's just one file.

After downloading the release ZIP file, it would be convenient to create a single HTML file from its contents.

### Example

```bash
$ make thereCanBeOnlyOne  #  output is a single HTML file that has everything
```

**Comment by n1474335** [comment]
As much as this is appealing, it unfortunately causes a lot of issues on the development side. CyberChef is a very complex web app now and we've come up against a number of issues trying to support the single file version. We made the decision earlier this year to end support for it and instead offer a zip file containing all the required modules and assets. This version is still standalone. It doesn't require a server or any kind of network connection. You just extract the zip and then open up the `.html` file in a decent browser.

We think this is a reasonable compromise as it makes development a lot simpler and allows us to press ahead with some of the new features we're working on without having to constantly develop workarounds. We employed a similar attitude when we decided not to support legacy browsers. Yes, it would be nice, but it would severely hamper our ability to push out new features and we value that more.
